### PR TITLE
Font Color (Dark view)

### PR DESCRIPTION
**Pull request by hcvcastro**
- **kit: fix missing theme option**
- **config: fix font color value (Dark Theme)**

### CODE REVIEW

File: kit/Kit.cpp [hunk 1]
```diff
@@ -1593,6 +1594,9 @@ class Document final : public DocumentManagerInterface
         if (!userTimezone.empty())
             options += ",Timezone=" + userTimezone;
```
Comment (caolanm) [inline kit/Kit.cpp:1596]: I get the first commit, seems very plausible

File: coolkitconfig.xcu [hunk 2]
```diff
@@ -604,7 +604,7 @@
         </node>
         <node oor:name="CalcText">
             <prop oor:name="Color" oor:op="fuse">
-                <value>15658734</value>
```
Comment (caolanm) [inline coolkitconfig.xcu:607]: But I don't really get the second commit. Do they have a known effect, I mean "CalcText" of 15658734 is 0xEEEEEE which is pretty close to 0xFFFFFF already, and "CalcValue" of 7512015 is a bright blue of 0x729FCF and I am unsure where they get used.

Do we need the second commit to fix the known issue?

Comment (hcvcastro) [reply coolkitconfig.xcu:607]: I did not test CalcText and CalcValue how is used, but I changed to be sure all white.

Comment (caolanm) [reply coolkitconfig.xcu:607]: Searching in coolkitconfig.xcu I see that the "Light" and "Dark" colors for "CalcText" and "CalcValue" are already quite different from eachother, while "FontColor" is 0 in both. I suggest to drop those two hunks about "Calc*" we're not sure what they affect, seeing as they are already quite different in Dark/Light they are probably ok.

Comment (hcvcastro) [reply coolkitconfig.xcu:607]: Ok, I will drop CalcText and CalcValue, but FontColor it fixes the Writer Text font color case.

Comment (caolanm) [reply coolkitconfig.xcu]: yeah, it looks odd the have the same "FontColor" in Light and Dark.

**Review by caolanm — APPROVED** [review]

**Pull request merged**